neptune_tensorboard/sync/internal/path_parser.py

```diff
@@ -0,0 +1,60 @@
+#
+# Copyright (c) 2019, Neptune Labs Sp. z o.o.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import os
+import re
+
+_EVENTS_FILE_PREFIX = "events.out.tfevents."
+
+
+def parse_path_to_experiment_name(path):
+    """
+    Parses a relative path to tensorflow event file to a reasonable neptune-compatible name
+
+    Arguments:
+        path(str): a path relative to entrypoint directory
+
+    Returns: a string representing sanitized project name based on the path, or "untitled-tensorboard" if name cannot
+        be determined
+    """
+    os.path.dirname(path)
+    experiment_name = os.path.dirname(path)
+    sanitized_experiment_name = re.sub(r'[^0-9A-Za-z_\-]', '_', experiment_name)
```

Comment:
According to: https://github.com/neptune-ml/neptune-tensorboard/issues/8#issuecomment-476776897 experiment name should be the name of the tensorboard run, so this replacing seems unnecessary.